libraries/AP_NavEKF2/AP_NavEKF2_Logging.cpp

```diff
@@ -269,3 +270,83 @@ void NavEKF2::Log_Write()
         }
     }
 }
+
+void NavEKF2::Log_Write_GSF(uint8_t _core, uint64_t time_us) const
+{
+    float yaw_composite;
+    float yaw_composite_variance;
+    float yaw[N_MODELS_EKFGSF];
+    float ivn[N_MODELS_EKFGSF];
+    float ive[N_MODELS_EKFGSF];
+    float wgt[N_MODELS_EKFGSF];
+
+        // @LoggerMessage: NKY0
+        // @Description: EKF2 Yaw Estimator States
+        // @Field: TimeUS: Time since system startup
+        // @Field: YC: GSF yaw estimate (rad)
+        // @Field: YCS: GSF yaw estimate 1-Sigma uncertainty (rad)
+        // @Field: Y0: Yaw estimate from individual EKF filter 0 (rad)
+        // @Field: Y1: Yaw estimate from individual EKF filter 1 (rad)
+        // @Field: Y2: Yaw estimate from individual EKF filter 2 (rad)
+        // @Field: Y3: Yaw estimate from individual EKF filter 3 (rad)
+        // @Field: Y4: Yaw estimate from individual EKF filter 4 (rad)
+        // @Field: W0: Weighting applied to yaw estimate from individual EKF filter 0 (rad)
```

Comment:
Is this weighting really in radians?  The units field doesn't seem to say so.

We've generally tended to not put the units into this descriptive text as it's redundant given the "units" field - and here you've got redundancy you'll soon have mismatches :-)